testathon/readme.md

```diff
@@ -0,0 +1,34 @@
+# testathon_data_prepare
```

Comment:
Lets add some example commands to this doc to add clarity for how to use this tool.